src/ncp/spinel.h

```diff
@@ -830,6 +830,23 @@ typedef enum
     /** Format: `L` (Read-only) */
     SPINEL_PROP_CNTR_RX_SPINEL_ERR     = SPINEL_PROP_CNTR__BEGIN + 302,
 
+    /// The number of unicast packets transmitted.
+    /** Format: `L` (Read-only) */
+    SPINEL_PROP_CNTR_TX_PKT_UNICAST        = SPINEL_PROP_CNTR__BEGIN + 303,
+
```

Comment:
I suggest having these added/defined within the RX and TX section counters. We have a different range of spinel property values for `RX_PKT`,  `TX_PKT`, `RX/TX_IP` and then `RX/TX_SPINEL`


- The packet TX counter go from `SPINEL_PROP_CNTR__BEGIN + 1` to `SPINEL_PROP_CNTR__BEGIN + 99` 
- The packet RX counters go from `SPINEL_PROP_CNTR_RX_PKT_TOTAL      = SPINEL_PROP_CNTR__BEGIN + 100` to currently  `    SPINEL_PROP_CNTR_RX_PKT_DUP        = SPINEL_PROP_CNTR__BEGIN + 114,`